### PR TITLE
Fb starting on UI

### PR DESCRIPTION
**Pull request by FadiBaker92**

### CODE REVIEW

File: lib/tasks/dev.rake
Comment (jrzricardo) [inline lib/tasks/dev.rake:1]: got the known users in, nice.

File: config/routes.rb
Comment (jrzricardo) [inline config/routes.rb:1]: root first, excellent. consider alphabetizing your resources, but not necessary.

File: app/views/shared/_navbar.html.erb
Comment (jrzricardo) [inline app/views/shared/_navbar.html.erb:1]: solid bar.

File: app/views/shared/_flash.html.erb
Comment (jrzricardo) [inline app/views/shared/_flash.html.erb:1]: nice flash.

File: app/views/shared/_cdn_assets.html.erb
Comment (jrzricardo) [inline app/views/shared/_cdn_assets.html.erb:1]: top notch copy/pasta

File: app/views/likes/index.html.erb
Comment (jrzricardo) [inline app/views/likes/index.html.erb:1]: good clean up.

File: app/views/layouts/application.html.erb
Comment (jrzricardo) [inline app/views/layouts/application.html.erb:1]: new flashes look good.

File: app/controllers/application_controller.rb
Comment (jrzricardo) [inline app/controllers/application_controller.rb:1]: they shall not pass. . . w/o a login

**Review by jrzricardo — APPROVED** [review]

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
+
+  before_action :authenticate_user!
```
Comment (JoseMDO) [inline app/controllers/application_controller.rb:3]: Nice!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,11 +6,27 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render partial: "shared/cdn_assets" %>
```
Comment (JoseMDO) [inline app/views/layouts/application.html.erb:9]: Great job adding your CDN assets to a partial!

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Resources
+          </a>
+          <div class="dropdown-menu">
+            <a class="dropdown-item" href="/photos">Photos</a>
```
Comment (JoseMDO) [inline app/views/shared/_navbar.html.erb:14]: could use link_to tags here now :)

**Review by JoseMDO — APPROVED** [review]
Looks good Fadi!

**Pull request merged**